packages/components/src/StatusIndicator/StatusIndicator.css

```diff
@@ -1,8 +1,12 @@
+:root {
```

Comment:
Moved this from statusLabel to the indicator as it's no longer referenced in that CSS just this one.

Reply:
Makes sense